docs/data/date-pickers/overview/overview.md

```diff
@@ -7,67 +7,19 @@ materialDesign: https://m2.material.io/components/date-pickers
 waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/datepicker-dialog/
 ---
 
-# MUI X Date and Time Pickers
-
-<p class="description">These react date picker and time picker components let users select date or time values.</p>
-
-{{"component": "@mui/docs/ComponentLinkHeader"}}
-
-## Overview
-
-{{"demo": "CommonlyUsedComponents.js"}}
-
-## Community or Pro plan?
-
-The Date and Time Pickers are available in two packages:
-
-- `@mui/x-date-pickers`, which is MIT licensed (free forever) and contains all the components to edit a date and/or a time.
-- `@mui/x-date-pickers-pro`, which is [commercially licensed](/x/introduction/licensing/#pro-plan) and contains additional components to edit date and/or time ranges.
-
-## Date library
-
-The Date and Time Pickers are focused on UI/UX and, like most other picker components available, require a third-party library to format, parse, and mutate dates.
+{{"component": "modules/components/overview/XLogo.tsx"}}
 
-MUI's components let you choose which library you prefer for this purpose.
-This gives you the flexibility to implement any date library you may already be using in your application, without adding an extra one to your bundle.
-
-To achieve this, both `@mui/x-date-pickers` and `@mui/x-date-pickers-pro` export a set of **adapters** that expose the date manipulation libraries under a unified API.
-
-### Available libraries
-
-The Date and Time Pickers currently support the following date libraries:
-
-- [Day.js](https://day.js.org/)
-- [date-fns](https://date-fns.org/)
-- [Luxon](https://moment.github.io/luxon/#/)
-- [Moment.js](https://momentjs.com/)
-
-:::info
-If you are using a non-Gregorian calendar (such as Jalali or Hijri), please refer to the [Support for other calendar systems](/x/react-date-pickers/calendar-systems/) page.
-:::
-
-### Recommended library
-
-If you are already using one of the libraries listed above in your application, then you can keep using it with the Date and Time Pickers as well.
-This will avoid bundling two libraries.
-
-If you don't have your own requirements or don't manipulate dates outside of MUI X components, then the recommendation is to use `dayjs` because it has the smallest impact on your application's bundle size.
-
-Here is the weight added to your gzipped bundle size by each of these libraries when used inside the Date and Time Pickers:
+# MUI X Date and Time Pickers
```

Comment:
```suggestion
# Date and Time Pickers
```

Perhaps we could drop the MUI X from the title now we have the badge, what do people think?

Reply:
I like the idea 🤔

Reply:
We probably have a small SEO topic here, cc @joserodolfofreitas 
Visually I fully agree that it make no sense to have the information twice

Reply:
We could solve the SEO issue by changing the markup a little, something like: https://codepen.io/KenanYusuf/pen/jOjVWVy

It's not the prettiest, but the text content would remain "MUI X Date and Time Pickers"

Reply:
That's a good point, @flaviendelangle. I'd be fine solving the SEO with something else, but for now let's keep the "MUI X" in the title. We can improve this on the following iterations.

Reply:
@noraleonte noticed that on this page we are already using a "title" property which equals `React Date Picker and Time Picker components`
So dropping "MUI X" shouldn't have any impact on the SEO here

Reply:
OK I'm not competent enough on this topic :grimacing: